triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -579,13 +606,6 @@ private[lf] class Runner private (
     completionSource merge transactionSource merge heartbeatSource
   }
 
-  private def logReceivedMsg(tm: TriggerMsg): Unit = tm match {
```

Comment:
In a future PR, `Ctx` based logging on the flow will render this logging useless - removing it now for simplicity